pkg/rancher-desktop/pages/Required.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div>
+    <h2>{{ t('Required.title') }}</h2>
+    <h3>{{ t('Required.message') }}</h3>
```

Comment:
It seems awkward to have a title, then the message, and then an explanation. I'd have just the title as it is, and then an explanation saying something along the lines of "Missing requirements: nested virtualization is not enabled on this host.

I'm not a UI oriented person though... @rak-phillip what's your take?

Screenshot for reference:

![Screenshot from 2022-12-16 11-28-07](https://user-images.githubusercontent.com/19395776/208165601-0372e13f-63e0-40c6-901b-2d51ca05962b.png)

Reply:
I agree - the subtitle doesn't quite fit in with the dialog. We should strive for consistency with other custom dialogs that we use throughout Rancher Desktop

![image](https://user-images.githubusercontent.com/835961/208181852-e4481e74-85b9-44d1-a612-e0be764f8ff6.png)

I quite like the approach of updating the content of the details to be something along the lines of what @adamkpickering recommends. We might be able to switch around some of the text depending on what's most important for us to communicate. Before providing more feedback, I'd like to better understand how we expect this to function. @dpmerrill are we able to flag multiple reasons for why Rancher Desktop was unable to start or is it just one?

Reply:
@rak-phillip it is currently just a single reason, but I was trying to leave it open for the possibility of more in future tests.
But now that I think about it, I think it should be a single reason because that meets our current needs. If we need to change it to a list in the future, the dialog can be re-worked then.

Reply:
I ultimately think we can keep the open to the possibility of having multiple reasons and keep the same design if we only have one. We could probably distill this down to something along the lines of: 

> ## Rancher Desktop is unable to start
>
> Rancher Desktop cannot start because requirements are missing or not configured:
> - Nested virtualization is not enabled on this host
>
> Please, ensure all requirements are met and try again. Rancher Desktop will now close.

I think that these dialogs would benefit from a help button. I added #3665 to address.

Reply:
changed